docs/faq.rst

```diff
@@ -0,0 +1,126 @@
+.. _faq:
+
+Frequently Asked Questions
```

Comment:
nit: Frequently asked questions

We've always only capitalized the first letter so far